python/plugins/db_manager/db_plugins/gpkg/connector.py

```diff
@@ -620,6 +620,14 @@ def runVacuum(self):
         """ run vacuum on the db """
         self._execute_and_commit("VACUUM")
 
+    def commentTable(self, schema, tablename, comment=None):
+        """Comment the table"""
+        return ''
```

Comment:
No need to return
```suggestion
        pass
```
but, IMO it would be cleaner if you checked for the connector to support comments before calling `commentTable` in `/python/plugins/db_manager/dlg_table_properties.py` instead of silently failing, it would be much more clear for whom read your caller code.